shared/common-adapters/oriented-image.desktop.js

```diff
@@ -0,0 +1,71 @@
+// @flow
+import * as React from 'React'
+import EXIF from 'exif-js'
+import {noop, inRange} from 'lodash-es'
+import {Image} from '../common-adapters'
+import {type Props} from './oriented-image.types'
+import {collapseStyles, type StylesDesktop} from '../styles'
+
+type State = {
+  styleTransform: StylesDesktop | null,
+}
+
+const makeStyleTransform = (orientation: number): StylesDesktop =>
+  exifOrientaionMap[orientation] || {transform: 'rotate(0deg)'}
```

Comment:
we might want to log if we get a different orientation and fallback to 0 so we can debug user logs

Reply:
What log level would be appropriate? warn?